src/components/CommandBar/CommandBar.tsx

```diff
@@ -110,6 +110,7 @@ export class CommandBar extends React.Component<ICommandBarProps, ICommandBarSta
                 role='menuitem'
                 aria-label={ this.props.elipisisAriaLabel || '' }
                 aria-haspopup={ true }
+                data-automationid='commandBarOverflow'
```

Comment:
instead of tacking this on, it would be ideal to just enable div props to be mixed into the root.